script/core

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# Helper to start Home Assistant Core inside the devcontainer
+
+# Stop on errors
+set -e
+
+if [ -z "${DEVCONTAINER}" ]; then
+  echo "This task should only run inside a devcontainer, for local install HA Core in a venv."
+  exit 1
+fi
+
+if [ ! -z "${CODESPACES}" ]; then
+  WORKSPACE="/root/workspace/frontend"
+else
+  WORKSPACE="/workspaces/frontend"
+fi
+
+if [ -z $(which hass) ]; then
+  echo "Installing Home Asstant core from dev."
+  python3 -m pip install --upgrade \
+    colorlog \
+    git+git://github.com/home-assistant/home-assistant.git@dev
+fi
+
+if [ ! -d "${WORKSPACE}/config" ]; then
+  echo "Creating default configuration."
+  mkdir -p "${WORKSPACE}/config";
+  hass --script ensure_config -c config
+  echo "demo:
+
+logger:
+  default: info
+  logs:
+    homeassistant.components.frontend: debug
+
+" >> "${WORKSPACE}/config/configuration.yaml"
+
+  if [ ! -z "${HASSIO}" ]; then
+  echo "
+# frontend:
+#   development_repo: ${WORKSPACE}
+hassio:
+  development_repo: ${WORKSPACE}" >> "${WORKSPACE}/config/configuration.yaml"
+  else
+  echo "
+frontend:
+  development_repo: ${WORKSPACE}
+# hassio:
```

Comment:
Let's add new lines or else one might mistake `hassio` to be a child underneath `frontend`
```suggestion

hassio:
  development_repo: ${WORKSPACE}" >> "${WORKSPACE}/config/configuration.yaml"
  else
  echo "
frontend:
  development_repo: ${WORKSPACE}

# hassio:
```